roles/ealapps/tasks/main.yml

```diff
@@ -7,6 +7,7 @@
     update: true
     force: true
```

Comment:
Could we add the .htaccess file at hrc/.htaccess as part of this role?

Here is what it should include: https://github.com/PrincetonUniversityLibrary/video_reserves/blob/c72cd038b8f5d6155aebee75468a75bdbd834604/hrc/.htaccess